mapswipe_workers/mapswipe_workers/base/base_project.py

```diff
@@ -221,7 +220,6 @@ def save_to_postgres(self, project, groups, groupsOfTasks):
             '''
```

Comment:
we also need to remove one `%s` in the line above.

Reply:
Removed placeholder. Good catch!